core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -103,6 +103,23 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
       }.recover(bigQueryRecover)
     }
 
+  def deleteEntityAttribute(workspaceName: WorkspaceName, entityType: String, entityAttributeNamespace: String, entityAttributeName: String, dataReference: Option[DataReferenceName], billingProject: Option[GoogleProjectId]): Future[PerRequestMessage] =
```

Comment:
So we wanted to make Entity Service be able to be generic across rawls entities and data repo snapshot data. So I added the `dataReference` and `billingProject` params in case we eventually wanted to implement this functionality for data repo snapshots too. But, that implementation would be so far off that it might not be useful to add this stuff in now.